### ISSUE TITLE
Issue with Prisma Cloud Compute 22.12

### ISSUE THREAD

**Issue by hi-artem** [PaloAltoNetworks/terraform-provider-prismacloudcompute#68]
## Describe the bug

@wfg I know we discussed it, but I am going to create this task just for the awareness of the community. Hopefully, someone would be interested to work on this.

The latest update to the Compute changed Container Runtime API that made this provider stop working for `prismacloudcompute_container_runtime_policy` resources.

## Expected behavior

Resource is created

## Current behavior

Apply fails:

```
Error: error updating containerRuntime policy: Non-OK status: 400
```

## Possible solution

Fix provider to support 22.12

## Steps to reproduce

1. Create a resource https://registry.terraform.io/providers/PaloAltoNetworks/prismacloudcompute/latest/docs/resources/container_runtime_policy
2. Plan and apply against console with version 22.12
3. See results

**Comment by nk-rao** [comment]
Thanks for posting, we are also facing the same issue. Lookslike prisma team is no longer maintaining this code. We may need to migrate to different solution

**Comment by pc-mrousseau** [comment]
We have the same issue as well and would like to see PA fix this.

**Comment by hi-artem** [comment]
@pc-mrousseau @nk-rao I would recommend to escalate this to your account manager or PAN support team. It will generate more attention. I don't think PAN realize how many customers are actually using this.

**Comment by maxboynton** [comment]
Issue has been addressed in the v0.8.0 release (https://github.com/PaloAltoNetworks/terraform-provider-prismacloudcompute/pull/71).
